mirgecom/discretization.py

```diff
@@ -38,28 +38,25 @@
 # we can replace it more easily when we refactor the drivers and
 # examples to use discretization collections, and change it centrally
 # when we want to change it.
-# TODO: Make this return an actual grudge `DiscretizationCollection`
-#       when we are ready to change mirgecom to support that change.
 def create_discretization_collection(actx, mesh, order, *, mpi_communicator=None,
                                      quadrature_order=-1):
-    """Create and return a grudge DG discretization object."""
+    """Create and return a grudge DG discretization collection."""
     from grudge.dof_desc import DISCR_TAG_BASE, DISCR_TAG_QUAD
-    from grudge.eager import EagerDGDiscretization
+    from grudge.discretization import DiscretizationCollection
     from meshmode.discretization.poly_element import (
         QuadratureSimplexGroupFactory,
-        default_simplex_group_factory
+        PolynomialRecursiveNodesGroupFactory
     )
 
     if quadrature_order < 0:
         quadrature_order = 2*order+1
 
-    discr = EagerDGDiscretization(
+    return DiscretizationCollection(
         actx, mesh,
         discr_tag_to_group_factory={
-            DISCR_TAG_BASE: default_simplex_group_factory(base_dim=mesh.dim,
-                                                          order=order),
+            DISCR_TAG_BASE: PolynomialRecursiveNodesGroupFactory(order=order,
```

Comment:
it's back!

Reply:
heh.   yes, your intuition is working well.